backend/src/main/java/com/google/minicrm/data/Currency.java

```diff
@@ -0,0 +1,34 @@
+package com.google.minicrm.data;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum Currency {
+    @SerializedName("INR ₹") INDIAN_RUPEE("INR"),
+    @SerializedName("BRL R$") BRAZILIAN_REAL("BRL"),
+    @SerializedName("MXN $") MEXICAN_PESO("MXN"),
+    @SerializedName("USD $") US_DOLLAR("USD");
```

Comment:
Shouldn't we be using a library for these currencies (i.e. javax.money) so we don't have to redo all of this currency work? Same with the languages.